ee/server/service/software_installers.go

```diff
@@ -1650,6 +1650,9 @@ func (svc *Service) softwareInstallerPayloadFromSlug(ctx context.Context, payloa
 	payload.UninstallScript = app.UninstallScript
 	payload.FleetMaintained = true
 	payload.MaintainedApp = app
+	if len(payload.Categories) == 0 {
```

Comment:
per discussion in design review today: if we get 0 categories for an FMA, we should set them back to the default values (which come from the manifest)